src/routes/blog/spyder-6-daniel/+page.md

```diff
@@ -0,0 +1,102 @@
+
+## Remote development architecture quality testing
+
+Once the Editor migration was moving along, helped review, test and QA the new remote development architecture.
+As part of that, he also developed a number of fixes and improvements to address the issues he found.
+From that, he [implemented UI/UX improvements](https://github.com/spyder-ide/spyder/pull/22303) for the graphical components used to create a remote connection, based on the issues he found.
+
+Furthermore, during his checks of remote connections were working on Windows, he discovered several errors with when running async code, which [he fixed](https://github.com/spyder-ide/spyder/pull/22137).
+Finally, he also encountered problems when trying to tunnel ports through SSH, which he [also solved](https://github.com/spyder-ide/spyder/pull/22223](https://github.com/spyder-ide/spyder/pull/22223).
+This all helped make our remote development architecture much more stable and usable.
+
+
+## Spyder test suite overhaul
+
+To help support testing of Spyder 6 and its new components, Daniel helped make the Spyder test suite faster and more robust.
+The core team was witnessing constant errors and delays when running automated tests in continuous integration systems, which made it necessary for Daniel invest several weeks to fix that so that the Spyder 6 development process was smoother.
+Therefore, [Daniel's work](https://github.com/spyder-ide/spyder/pull/22077) improved this situation, as well as making the tests more efficient overall.<br><br>
+
+
+![Spyder's file switcher dialog, with a path box to enter a file name above a list of matching files and their properties](file-switcher-improvements.png "Spyder's File Switcher, with the improvements from Daniel and Angela's work")
+
+## Review work and project management
+
+Much of Daniel's responsibilities involve reviewing, mentoring and assisting with the work of other developers, an important role in any project.
+Daniel helped [Ryan Clary](https://github.com/mrclary), the developer who created a new set of installers for Spyder 6 using [Conda-Constructor](https://conda.github.io/constructor/), which are more powerful, flexible and updateble, and support installing packages, environments and Spyder plugins locally and remotely, as needed for the new remote development functionality among others.
+In addition, he tested and validated the installers' update process, as well as finding and reporting several bugs on Windows.
+
+Daniel also reviewed and assisted with [Angela Remolina](https://github.com/AngelaRemolina)'s work on major improvements to Spyder's File/Symbol Switcher.
+Angela, an intern who worked for on Spyder over several months, migrated the File/symbol switcher to a plugin, so it is modular and accessible to other plugins.
```

Comment:
```suggestion
Angela, an intern who worked on Spyder over several months, migrated the File/Symbol switcher to a plugin, so it is modular accessible to other plugins.
```

* `for` seems unnecessary.
* `File/Symbol` are both capitalized in the previous sentence.